CHANGELOG.md

```diff
@@ -2,10 +2,51 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.9.0
+# 10.9.0 October 21, 2022
```

Comment:
Date must be removed if you plan to use release train

Reply:
Oh, didnt know that, thanks

Reply:
Removed

Reply:
@yunikkk @kiryldz this applies to the PR towards main branch as well, it will be automatically generated and ported to main/release branch.

Reply:
btw thats why I think we should create this PR after the main branch PR is merged, otherwise we have diverged commits.

Reply:
> btw thats why I think we should create this PR after the main branch PR is merged, otherwise we have diverged commits.

Well the content is still the same so no conflicts will appear - and hashes will anyway be different

Reply:
> Well the content is still the same so no conflicts will appear - and hashes will anyway be different

The issue is that the commit message will also be different, and commit content will risk being diverged, imo it's better to just cherry-pick the merged commit, will be easier to find the exact same commit when comparing different branches.